includes/admin/class-wc-admin-attributes.php

```diff
@@ -440,6 +440,7 @@ public static function add_attribute() {
 							<h2><?php _e( 'Add New Attribute', 'woocommerce' ); ?></h2>
 							<p><?php _e( 'Attributes let you define extra product data, such as size or colour. You can use these attributes in the shop sidebar using the "layered nav" widgets. Please note: you cannot rename an attribute later on.', 'woocommerce' ); ?></p>
 							<form action="edit.php?post_type=product&amp;page=product_attributes" method="post">
+								<?php do_action( 'woocommerce_add_attribute_top' ) ?>
```

Comment:
Don't these hooks need to be in the Edit Attribute form too? (~lines 272 and 343)